graphein/ml/datasets/pdb_data.py

```diff
@@ -1150,6 +1304,81 @@ def select_complexes_with_grouped_molecule_types(
         if update:
             self.df = df
 
+    def has_uniprot_id(
+        self,
+        splits: Optional[List[str]] = None,
```

Comment:
Do you think it makes sense to have an additional arg containing a list of IDs to filter on for these methods?

Reply:
yes, that makes sense. Just added a version of that